package.nix

```
@@ -180,7 +185,8 @@ goBuild {
 
   preBuild = ''
     # build llama.cpp libraries for ollama
-    make -j 5
+    threads="$(nproc)"
+    make -j $((threads / 2))
```

I'm now starting to wonder whether `threads / 2` is too low. Do you think it should be higher?

```suggestion
    make -j $((threads * 3 / 4))
```